_episodes/04-ome-public-example.md

```diff
@@ -13,158 +13,147 @@ OME have published (and maintain) a set of example playbooks which use the same
 
 Here we're going to execute one of those examples - the "public user" example.
 
-> ## Don't worry!
->
-> Most of the following commands include some BASH shell expansions to simplify connecting to a Vagrant VM from different machines. It's really not as complicated as this makes it look.
->
-{: .callout}
-
-1 Acquire a copy of the examples (you may have done this already):
+If you haven't aleady cloned the GitHub repository with the deployment examples:
 
 ~~~
-$ git clone https://github.com/ome/ansible-examples-omero.git 
-# or
-$ git clone git@github.com:ome/ansible-examples-omero.git
+$ git clone --recurse https://github.com/ome/omero-deployment-examples
 ~~~
 {: .bash} 
 
-Change directory to the 'public-user' example in the clone of ansible-examples-omero
+Change to the `ansible-example-omero-public-user` directory in the clone of `omero-deployment-examples`:
 ~~~
-$ cd ansible-examples-omero/public-user
+$ cd omero-deployment-examples/ansible-example-omero-public-user/
 ~~~
 {: .bash} 
 
-Install the prerequisite roles, defined in requirements.yml
+Install the prerequisite roles, defined in [`requirements.yml`](https://github.com/ome/ansible-example-omero-public-user/blob/4ed8a3ba4aee4ab409f6905add7e23aaf17b7f98/requirements.yml):
 ~~~
-$ ansible-galaxy install -r ../requirements.yml -p roles
+$ ansible-galaxy install -r requirements.yml -p roles
 ~~~
 {: .bash}
 
-Add some port forwarding to the Vagrantfile to allow us to connect to OMERO afterwards - and also remove the "provision" step - since we're learning here how to do this ourselves.
+The Vagrantfile will create a CentOS 7 virtual machine with port-forwarding so you can access the deployed server.
+
 ~~~
-$ rm Vagrantfile && wget https://gist.github.com/kennethgillen/648105ba0f78440ca41e45963c471744/raw/Vagrantfile
+$ cat Vagrantfile
 ~~~
 {: .bash} 
+~~~
+# -*- mode: ruby -*-
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "centos/7"
+
+  config.vm.provider "virtualbox" do |vb|
+    config.vm.network "forwarded_port", guest: 80, host: 8080, auto_correct: true
+    config.vm.network "forwarded_port", guest: 443, host: 8443, auto_correct: true
+    config.vm.network "forwarded_port", guest: 4063, host: 4063, auto_correct: true
+    config.vm.network "forwarded_port", guest: 4064, host: 4064, auto_correct: true
+  end
+
+  config.vm.provider "virtualbox" do |vb|
+    vb.memory = "2048"
+    vb.cpus = "2"
+  end
+
+  # Uncomment this to automatically run Ansible
```

Comment:
Could also be updated to match https://github.com/ome/ansible-example-omero-public-user/pull/3